docs/mappings/README.md

```diff
@@ -0,0 +1,9 @@
+# Gallery of Mappings
+
+This directory contains a "gallery" of sample OpenLineage to Purview Mappings that can be used in the `OlToPurviewMappings` App Setting on the solution's Azure Function.
+
+## Azure SQL
+
+* [Prioritize Azure SQL Non DBO](./az-sql-non-dbo.json)
```

Comment:
Can we add the azureSQLNonDboNoDotsInNames section to the json and change the name of this file to az-sql.json?  Seems like we will have a per source set of configurations in this directory.  I can add the rest.